subsys/bluetooth/host/audio/mcc.c

```diff
@@ -1758,6 +1824,37 @@ int bt_mcc_read_current_track_obj_id(struct bt_conn *conn)
 	return err;
 }
 
+int bt_mcc_set_current_track_obj_id(struct bt_conn *conn, uint64_t obj_id)
+{
+	int err;
+
+	if (!conn) {
+		return -ENOTCONN;
+	}
+
+	if (!cur_mcs_inst->current_track_obj_id_handle) {
+		BT_DBG("Handle not set");
+		return -EINVAL;
+	} else if (cur_mcs_inst->busy) {
+		return -EBUSY;
+	}
+
+	sys_put_le48(obj_id, cur_mcs_inst->write_buf);
+	cur_mcs_inst->write_params.offset = 0;
+	cur_mcs_inst->write_params.data = cur_mcs_inst->write_buf;
+	cur_mcs_inst->write_params.length = BT_OTS_OBJ_ID_SIZE;
+	cur_mcs_inst->write_params.handle = cur_mcs_inst->current_track_obj_id_handle;
+	cur_mcs_inst->write_params.func = mcs_write_current_track_obj_id_cb;
+
+	BT_HEXDUMP_DBG(cur_mcs_inst->write_params.data, BT_OTS_OBJ_ID_SIZE, "Object Id");
```

Comment:
Consider using `bt_hex` instead, as they will also log the function name
```suggestion
	BT_DBG("Object ID: %s", bt_hex(cur_mcs_inst->write_params.data, BT_OTS_OBJ_ID_SIZE));
```

Reply:
Added to later task